SU-SWS/ace-sdssgryphon
# SDSS-475: removal of the text from brandbar

# READY FOR REVIEW

# Summary
- Removal of the text from brandbar
- Removal of the SCSS for that text

# Review By (Date)
- 1/31

# Urgency
- High

# Steps to Test

1. Checkout the this code.
2. Switch a header to use option a or b
3. Verify brandbar text gone.
4. Verify it in multiple resolutions.

# Affected Projects or Products
- SDSS subtheme

# Associated Issues and/or People
[- SDSS-475](https://stanfordits.atlassian.net/browse/SDSS-475)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
